src/application.rs

```diff
@@ -2,8 +2,8 @@
 
 use abscissa::{self, Application, LoggingConfig};
 
-use commands::KmsCommand;
-use config::KmsConfig;
+use crate::commands::KmsCommand;
+use crate::config::KmsConfig;
```

Comment:
Depending on how much you care it might be nice to collapse these into nested imports (I've been doing that in my own crates), e.g.

```rust
use crate::{
    commands::KmsCommand,
    config::KmsConfig
};
```